libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Input/AttachmentInput.cs

```diff
@@ -51,6 +59,12 @@ public AttachmentInput([CallerFilePath] string callerPath = "", [CallerLineNumbe
         [JsonProperty("outputFormat")]
         public EnumExpression<AttachmentOutputFormat> OutputFormat { get; set; } = AttachmentOutputFormat.First;
 
+        /// <summary>
+        /// Called when input has been received, override this method to customize recognition of the input.
```

Comment:
Since this is already an override, I'm not sure the part about overriding it should be there.